README.md

```diff
@@ -405,12 +410,22 @@ Error: Try 'puppet help module install' for usage
 For Puppet 4.2.2+ (and 3.8.2) we've decreased the possibility of the issue occurring based on the fixes in [PUP-4854](https://tickets.puppetlabs.com/browse/PUP-4854). A complete fix will become available in a future version of Puppet that incorporates [PUP-4866](https://tickets.puppetlabs.com/browse/PUP-4866).
 If you are affected by this issue, you can work around this by downloading the `.tar.gz` from the Forge and use `puppet module install` using the file rather than directly from the Forge.
 
-- Windows 2003 is not supported. **If this module is present on the master, it breaks Windows 2003 agents.** 
+- Windows 2003 is not supported. **If this module is present on the master, it breaks Windows 2003 agents.**
 
   When installed on a Puppet master to the default `production` environment, this module causes pluginsync to **fail** on Windows 2003 agents because of an issue with [LFN (long file names)](https://tickets.puppetlabs.com/browse/PUP-4866). To work around this issue, host your Windows 2003 nodes on a [Puppet environment](https://docs.puppetlabs.com/puppet/latest/reference/environments.html) that is separate from `production` and that does **not** have the DSC module installed.  
-  
+ - Windows 2008 is not supported. Microsoft didn't publish WMF 5.0 RTM at 2016-02-24.
```

Comment:
Guess this was never supported. Some folks may be confused by this statement, so it's likely better to state `Windows 2008 R1` to reduce that confusion.